### ISSUE TITLE
Check for expired jobs

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Future#232]
Is there a way to check for jobs that have passed their date without running?
![Screen Shot 2022-03-24 at 12 03 29 PM](https://user-images.githubusercontent.com/3868028/159959522-5f900151-76f1-4d5e-bd0f-d34d86247578.png)

**Comment by andergmartins** [comment]
Duplicating #229

**Comment by andergmartins** [comment]
@stevejburge maybe something like this?

![image](https://github.com/user-attachments/assets/537cf40c-623c-48e4-b8d0-b2684dd44bca)

**Comment by stevejburge** [comment]
@andergmartins Sorry, this is a very old issue. I think it's now solved by our action logs?

**Comment by andergmartins** [comment]
Not really. The log is only available when debug is enabled, which rarely happens on production sites. It's already implemented as the attached image. Maybe include it anyway?

**Comment by stevejburge** [comment]
Sure thing, go for it
